packages/react-app/src/components/bridge/ToToken.jsx

```diff
@@ -102,7 +141,9 @@ export const ToToken = () => {
               <Text fontSize="lg" fontWeight="bold">
                 {token.name}
               </Text>
-              <AddToMetamask token={token} ml="0.5rem" asModal />
+              {token.address !== ADDRESS_ZERO && (
```

Comment:
This check shouldn't be necessary cause AddToMetamask has the check inside the component. It returns `null` if token.address is ADDRESS_ZERO